### ISSUE TITLE
Can't make the audio work

### ISSUE THREAD

**Issue by tremblerz** [ernw/insinuator-snippets#4]
I tried to follow all of the given procedures but Audio didn't work.

**Comment by 0xb35c** [comment]
Hi,
you are talking about WebEx, right?
If so, what OS do you use? It might also be that the libraries are at another place, though I'd assume that the plugin won't launch at all. Also, are you sure you are using the x86 bit versions of both JRE and Firefox?

**Comment by tremblerz** [comment]
Yes. I have been trying to run webex. My OS is Ubuntu 14.04 and yes I tried with all 4 possible combinations of x86 and x64.
One more thing I noticed is that if the path specified for JRE is wrong it doesn't throw any error nor it stops.

**Comment by 0xb35c** [comment]
The issue is that the audio on the 64bit java libs don't work with webex. If you specified a wrong path it should simply use the 64bit libs and still "work".

Anyway. When you have launched webex, i.e. the java plugin, can you check how it is called? You can e.g. use htop for that to see the command line params. Look for the firefox plugin-container. For me it contains the path to the IcedTeaPlugin.so, which will be used. For me this is not even in my home dir.

**Comment by 0xb35c** [comment]
@tremblerz Were you able to resolve this? If so please close the issue. Thanks
